helpdesk_mgmt/demo/helpdesk_demo.xml

```diff
@@ -5,37 +5,28 @@
         <field name="groups_id" eval="[(4,ref('group_helpdesk_user'))]" />
     </record>
     <!-- Categories -->
-    <record id="helpdesk_category_1" model="helpdesk.ticket.category">
+    <record id="helpdesk_category_1_demo" model="helpdesk.ticket.category">
```

Comment:
Don't do these XML-IDs changes. They only add noise and it's not so relevant. It's the same as views.

Reply:
Fine, makes sense.